bootstrap

```diff
@@ -0,0 +1,3 @@
+#! /bin/sh
+set -x
+autoreconf -ivf --warnings=all
```

Comment:
Needs $1 to build out of tree.

Reply:
I'm not sure what you're asking for; the autoreconf step cannot be done outside of the tree.

Reply:
$ rm -rf temp
$ mkdir temp
$ cd temp
$ autoreconf -isvf ../abcip
autoreconf: Entering directory `../abcip'
autoreconf: configure.ac: not using Gettext
autoreconf: running: /opt/local/bin/aclocal --force -I m4
autoreconf: configure.ac: tracing
autoreconf: running: /opt/local/bin/glibtoolize --force
...
configure.ac:16: installing 'build-aux/compile'
configure.ac:12: installing 'build-aux/missing'
src/app/Makefile.am: installing 'build-aux/depcomp'
autoreconf: Leaving directory `../abcip'
$ ../abcip/configure 
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... ../abcip/build-aux/install-sh -c -d
...

Reply:
But why would you run autoreconf from a different directory?  It is no different than running it in the original directory.  The autoreconf script takes the last argument, pulls the directory name out of it, and then chdirs to it before running its subprograms.  I can add the $1, I'm just curious.

Reply:
Because it is easy.  Why does autoreconf support that?